Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -783,17 +820,28 @@ protected override void ProcessRecord()
                 contentStream = new MemoryStream(contentBuffer);
             }
 
+            // Get the existing storage object so we can use its metadata. (If it does not exist, we will fall back to
+            // default values.)
+            Object existingGcsObject = null;
+            Dictionary<string, string> existingObjectMetadata = null;
```

Comment:
The "= null;" is not redundant. It actually means something different. A declared variable may not be read from until it has been initialized. The compiler will throw an error if you fail to initialize existingGcsObject on all code paths.